ons_alpha/jinja2/templates/base.html

```diff
@@ -90,11 +90,17 @@
     {% set body_class = '' %}
 {% endif %}
 
+{%- set page_title -%}
+    {%- with current_site=wagtail_site() -%}
+        {% if current_site and page.pk == current_site.root_page.pk and current_site.site_name %}{{ current_site.site_name }} - {% endif %}{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.display_title | default(page.title) }}{% endif %}{% if current_site and page.pk != current_site.root_page.pk and current_site.site_name %} - {{ current_site.site_name }}{% endif %}
```

Comment:
I wonder if we should give other page types a display_title in order to avoid having to check every time for its existence and then fall back on the page title?

Reply:
There is not much we can do here; most times, it would be a title as it is mandatory. The default was a shorthand of doing another if statement which we do to check if they have an seo title instead.

Reply:
OK, sounds good to leave it as is.